pkg/api/graph/graph.go

```diff
@@ -122,7 +122,7 @@ var DefaultNamer Namer = namer{}
 func (namer) ResourceName(obj interface{}) string {
 	switch t := obj.(type) {
 	case uniqueNamer:
-		return t.UniqueName()
+		return t.UniqueName().UniqueName()
```

Comment:
Yeah, I know ... compile error otherwise.  That second UniqueName() could be a String() as well.  I'll admit I left it like this to generate some discussion.